reference-stores/nuxt/components/product/ProductCard.vue

```diff
@@ -1,22 +1,42 @@
 <template>
   <div @mouseenter="handleHover(true)" @mouseleave="handleHover(false)">
     <div
-      class="w-full aspect-w-1 aspect-h-1 bg-gray-200 rounded-lg overflow-hidden xl:aspect-w-7 xl:aspect-h-8"
+      class="
+        w-full
+        aspect-w-1 aspect-h-1
+        bg-gray-200
+        rounded-lg
+        overflow-hidden
+        xl:aspect-w-7 xl:aspect-h-8
+      "
     >
-      <nuxt-link :to="`/products/${product.content.handle}`" class="group">
-        <nuxt-picture
+      <nuxt-link :to="`/products/${product.content.handle}`">
+        <nuxt-img
+          class="w-full h-full object-cover"
           :src="product.content.featuredMedia.src"
           :alt="product.content.featuredMedia.altText"
           sizes="sm:100vw lg:50vw xl:25vw"
           quality="80"
-          class="picture"
+          fit="cover"
         />
       </nuxt-link>
```

Comment:
Because the only content of these cards is an image, these links have no accessible name. They also have no discernible focus styles other than the corners of the images going from curved to sharp, which isn't really enough to notice. 


This is a really good clickable card pattern, but happy to discuss other patterns that might work: 


https://inclusive-components.design/cards/

Reply:
Should be all set now @mdarrik

Reply:
The image links still lack focus styles.
I also noticed there's no way to get to the "add to cart" button without hover, which causes an issue for keyboard & mobile users.

Reply:
I'm thinking it might make sense to create separate tickets in the backlog around accessibility.  There are likely more instances that need improvement.  Might help focus the task more.  Thoughts?  @NWRichmond @mdarrik

Reply:
Absolutely should. I just wanted to catch these since you were making changes in the code in that location.

Reply:
@mdarrik should be all set now

Reply:
Thanks so much!